packages/alfa-rules/src/common/predicate/is-clipped.ts

```diff
@@ -37,19 +41,27 @@ export function isClipped(
               )
             ),
             // Or its parent is clipped
-            (node: Node) =>
-              node
-                .parent({
-                  flattened: true,
-                  nested: true,
-                })
-                .some(isClipped(device, context))
+            hasRelevantParent(device, isClipped(device, context))
           ),
           node
         )
       );
+  };
 }
 
+function hasRelevantParent(
+  device: Device,
+  predicate: Predicate<Node>
+): Predicate<Node> {
+  return (node) =>
+    (and(isElement, not(isPositioned(device, "static")))(node)
+      ? getOffsetParent(node, device)
+      : node.parent({
+          flattened: true,
+          nested: true,
+        })
+    ).some(predicate);
+}
```

Comment:
It seems that the other place where we do use relevant parent is in R83 where we always have the construction `getRelevantParent(…).some(predicate)`.
So we should probably do the following refactoring:
* Add in `common/expectation/` a `getPositionningParent` which does the full branching based on `position`.
* Add in `common/predicate` a `hasPositionningParent` which calls it and does the `.some(predicate)` bit.
* Use `hasPositionningParent` here and in R83, deleting the `RelevantParent` stuff in both places.